package.json

```diff
@@ -81,9 +81,9 @@
     "compile": "hardhat compile",
     "deploy": "scripts/predeploy && hardhat migrate",
     "deploy-ganache": "yarn deploy -- --force",
-    "deploy-ganache-manual": "yarn deploy -- --network ganache --force",
-    "deploy-hardhat": "yarn deploy -- --network hardhat --force",
-    "deploy-rinkeby": "yarn deploy -- --force --network rinkeby",
+    "deploy-ganache-manual": "yarn deploy -- --force --network ganache",
+    "deploy-hardhat": "yarn deploy -- --force --network hardhat",
```

Comment:
What if --graph-config is not passed? is it storing it in any place by default or just not storing it?